coordinator/src/metrics.rs

```diff
@@ -76,6 +80,9 @@ pub fn collect(node: Node) {
     position_metrics(&cx, &node);
 
     let inner_node = node.inner;
+    if let Ok(dlc_channels) = inner_node.list_dlc_channels() {
+        DLC_CHANNELS_AMOUNT.observe(&cx, dlc_channels.len() as u64, &[]);
```

Comment:
It would be nice if we could count the dlc channels in different states as well e.g. healthy dlc channels are Signed and OffchainClosed

OnChainClosed should not be part of the metric I guess

And all others are unhealthy.

Reply:
Can do, I would want to avoid recording all combinations though because this explodes in total metrics. 
Can you state which states you want to track?